.github/workflows/test.yml

```diff
@@ -43,3 +46,4 @@ jobs:
             npm install
             npm run build
           dist: public
+          teardown: 'true'
```

Comment:
```suggestion
          teardown: true
```

Should it be `true`?

Reply:
unfortunately actions don't accept booleans there

<img width="804" alt="Screenshot 2020-11-04 at 11 07 32" src="https://user-images.githubusercontent.com/6108793/98097989-0b609380-1e8e-11eb-8510-4c16a1a0dc1c.png">